aten/src/ATen/native/cuda/UnaryOpsKernel.cu

```diff
@@ -40,6 +40,15 @@ void ceil_kernel_cuda(TensorIterator& iter) {
   });
 }
 
+void expm1_kernel_cuda(TensorIterator& iter) {
+  AT_DISPATCH_FLOATING_TYPES_AND_HALF(iter.dtype(), "expm1_cuda", [&]() {
+    gpu_kernel(iter, []GPU_LAMBDA(scalar_t a) -> scalar_t {
+      return ::expm1(a);
```

Comment:
Could you verify that this is overloaded on CUDA?

Reply:
Yes, quoting the CUDA [Math API reference](https://docs.nvidia.com/cuda/cuda-math-api/group__CUDA__MATH.html#group__CUDA__MATH): "Note that floating-point functions are overloaded for different argument types"